config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'users#index'
+
+  resources :users, only: [:show] do
```

Comment:
- [ ] Nice work using resources for your routes, however, the `:users` route should include `index` and ` show`, kindly consider adding `index` to this route